src/diffusers/models/autoencoders/autoencoder_tiny.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         encoder_block_out_channels: Tuple[int, ...] = (64, 64, 64, 64),
         decoder_block_out_channels: Tuple[int, ...] = (64, 64, 64, 64),
         act_fn: str = "relu",
+        upsample_fn: str = "nearest",
```

Comment:
Is it the original default value of the function associated?

Reply:
yes